src/app/hooks/quota.ts

```diff
@@ -28,10 +31,10 @@ export const useQuota = (productId: ProductType) => {
     }, [config?.ams.apiBasePath, auth]);
 
     const getQuotaTypesByProductId = () => {
-        const {
-            ams: amsConfig
-        } = config || {};
-        const { kasQuotaId, kasTrialQuotaId, srsQuotaId, srsTrialQuotaId } = amsConfig || {};
+        const kasQuotaId = services.kafka.ams.instanceQuotaId
```

Comment:
Where is `services` coming from? I can't see it declared in this file

Reply:
I got project running now (after your help and doing some final improvements) including fixing that problem.
Need couple minutes